README.md

```diff
@@ -10,3 +10,19 @@ A python implementation of [web3.js](https://github.com/ethereum/web3.js)
 * Python 2.7, 3.4, 3.5 support
 
 Read more in the [documentation on ReadTheDocs](http://web3py.readthedocs.io/)
+
+## Developer setup
+
+If you would like to hack on web3.py, set up your dev environment with:
+
+```
+sudo apt-get install libssl-dev
+# ^ This is for Debian-like systems. TODO: Add more platforms
+
+git clone git@github.com:pipermerriam/web3.py.git
+cd web3.py
+virtualenv venv
+. venv/bin/activate
+pip install -r requirements-dev.txt
+python setup.py install
```

Comment:
Sure thing. Just to be clear, the `pip install -e .` ask overrides the `python setup develop` one, right?

As I understand, the first makes the second unnecessary.